addons/mail/models/res_partner.py

```diff
@@ -18,6 +18,9 @@ class Partner(models.Model):
     _inherit = ['res.partner', 'mail.activity.mixin', 'mail.thread.blacklist']
     _mail_flat_thread = False
 
+    email = fields.Char(string='Email', tracking=1)
+    phone = fields.Char(string='Phone', tracking=2)
```

Comment:
For `tracking` we have no choice but don't override the `string` in mail though, also why do need to set it explicitly?